test/utilities.jl

```diff
@@ -60,6 +59,7 @@ end
 end
 
 @testset "PCA" begin
+	io = IOBuffer()
```

Comment:
I have a different fix for this in #427, but let's go with this version for now. :smile:

Reply:
I just saw that when I was looking through #427